swagger.json

```diff
@@ -2017,6 +2072,133 @@
         "label",
         "hasChildren"
       ]
+    },
+    "ConceptMapping": {
+      "type": "object",
+      "properties": {
+        "uri": {
+          "type": "string",
+          "description": "URI of the concept"
+        },
+        "notation": {
+          "type": "string",
+          "description": "Notation of the concept"
+        },
+        "type": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "concepts type, e.g. \"skos:exactMatch\"."
```

Comment:
I think this should be "mapping type" not "concepts type"

Reply:
Changed to mapping type